apps/emqx/src/emqx_channel.erl

```diff
@@ -3329,6 +3363,13 @@ disconnect_attrs(Reason, Channel) ->
     emqx_external_trace:disconnect_attrs(Reason, Channel).
 -endif.
 
+mk_common_hook_context(Channel) ->
+    Session = Channel#channel.session,
+    #{
+        chan_info_fn => fun(Prop) -> emqx_channel:info(Prop, Channel) end,
```

Comment:
avoid anonymous func.
`{fun ?MODULE:info/2, [Channel]}`

Reply:
actually make sense to use anonymous func, otherwise may lead to a large term for reporting.
e.g. in crash report, or if run process_info/0 from remote_console.